### DISCUSSION TITLE
Save on Exit

### DISCUSSION THREAD

**Discussion by Captain-Leftovers** [remotely-save/remotely-save#179] [Ideas]
is it possible to implement something like save on exit
maybe not after exit but somehow to signal to sync/update before it closes

**Comment by fyears** [comment]
sorry but no, because the plugin isn't aware of the obsidian exit. no way to work around that

**Comment by BananaLumps** [comment]
A few years later and we may be able to to it now.
https://docs.obsidian.md/Reference/TypeScript+API/Workspace/on('quit')

Does this help in any way to achieve this goal now, I too would like to see this feature added.
